showcases/shared/radio.json

```
@@ -156,5 +156,27 @@
 				}
 			}
 		]
+	},
+	{
+		"name": "Show Required Asterisk",
```

```suggestion
		"name": "Show asterisk for required form fields (radio buttons)",
```

I'm struggling whether it might be easier to understand when explaining it with some more words.

---

https://github.com/db-ux-design-system/core-web/pull/4509#discussion_r2196900438